torchvision/prototype/transforms/_geometry.py

```diff
@@ -40,13 +40,21 @@ class Resize(Transform):
     def __init__(
         self,
         size: Union[int, Sequence[int]],
+        # Unrelated to V2, just interested in your thoughts on this:
+        # It'd be nice to just be able to do Resize(mode="bilinear"), i.e. accept strings.
+        # Having to import the Enum class can be inconvenient when prototyping,
+        # and the discrepancy between torchvision and torch core (which accept
+        # strings) also trips users.
+        # I don't remember if this was supported in the past and then removed?
```

Comment:
@NicolasHug no it wasn't supported. [In this PR](https://github.com/pytorch/vision/pull/2952#pullrequestreview-524098958) we moved from PIL resampling int values to enums. It was also proposed to choose between : 1) `str` values or 2) Enums. I do not quite remember why we went to Enum instead of str. Most probably for the following reasons:

>  using enums instead of raw values has multiple benefits (less mistakes, clearer API, easier to search usages etc)